Chapters/1-2-Chapter2.md

```diff
@@ -0,0 +1,125 @@
+## Chapter 2
+
+Working in the field never felt lonely to me, though I knew some who resented it for that reason. My thoughts may betray me sometimes, but never out there. There was far too much to be done to have thoughts of other places and other times.
+
+Every morning was a repeat of the same movements I had learned from countless mornings of dressing with my mother. I would descend to the cellar to dress for the long day of sowing or harvesting or maintenance. The smell of damp on the stone walls pervaded the air and reminded me that my mother was with me. I would hook a lightglass to the overhead electrical rails, and little by little the flow of electricity grew and warmed the metal ribbon inside to a bright white. Drying on a length of twine were a pair of canvas trousers, a thickly-woven tunic, and gloves stained with clay and chlorophyll. No matter how often I dirtied them or dinged them up, they withstood it all for thousands of days. I was certain that my mother knew how to make the most durable clothes of anyone in Heiko.
+
+We used to grow pohua, a hardy root vegetable known for its sweet and fruity taste. As a toddler, my mother would carry me in a big thatch bag on her back while thi tilled, planted, harvested. I watched hae hands unfold those tiny tubers across the clay and pack them in. At days end, I would gaze in awe of how much thi could plant and harvest. It's been sixty-five crop rotations since thi passed, but thi lives as vividly in my memories as thi did in life.
```

Comment:
> It's been sixty-five crop rotations since thi passed, but thi lives as vividly in my memories as thi did in life.

this sentence feels like a mouthful. What about just: "It's been sixty-five crop rotations since thi passed, but thi still lives as vividly in my memories."